core/src/HPX/Kokkos_HPX.hpp

```diff
@@ -790,8 +790,16 @@ class TeamPolicyInternal<Kokkos::Experimental::HPX, Properties...>
     m_league_size           = league_size_request;
     const int max_team_size = 1;  // TODO: Can't use team_size_max(...) because
                                   // it requires a functor as argument.
-    m_team_size =
-        team_size_request > max_team_size ? max_team_size : team_size_request;
+
+    if (team_size_request > max_team_size) {
+      std::stringstream error;
+      error << "Requested too large team size. "
+               "Requested: "
+            << team_size_request << ", Maximum: " << max_team_size;
```

Comment:
To be coherent with what is done for Cuda, HIP and SYCL
```suggestion
      error << "Kokkos::TeamPolicy<HPX>: Requested too large team size. "
               "Requested: "
            << team_size_request << ", Maximum: " << max_team_size;
```